apps/web/lib/features/team/user-team-card/task-skeleton.tsx

```diff
@@ -61,15 +61,15 @@ export function UserTeamCardHeader() {
 					<div className="w-4 self-stretch border-l-[0.125rem] border-l-transparent " />
 					<div className="2xl:w-48 3xl:w-[12rem] w-1/5 lg:px-4 !pl-6 lg:!pl-8  flex flex-col items-center text-center  justify-center">
 						<Tooltip label={t('task.taskTableHead.WORKED_ON_TASK_HEADER_TOOLTIP')}>
-							{t('task.taskTableHead.TASK_WORK.TITLE')} <br /> {t('common.TASK')}
+							{t('common.TASK')} Time
```

Comment:
@CREDO23 hm, why we are doing this change? Now "Time" become hardcoded and will NOT be translated!

Reply:
Let me translate